_source/_posts/2021-01-08-graphql-apollo-react.md

```diff
@@ -28,19 +28,16 @@ If you prefer to watch the demo, you can view this walk through.
 
 ## Create your Okta Application
 
-The first thing you will need a free developer account from Okta. if you don't have one you can [sign up for one here.](https://developer.okta.com/signup/). Once you have completed that navigate to your developer's console and click on **Applications** and then **Add Application**. Select **Single Page App** and click **Next**.   
+{% include setup/cli.md type="spa" framework="React" loginRedirectUri="http://localhost:3000/callback" %}
 
-Give your application a meaningful name. You will also need to change your URIs to *localhost:3000* as this is the default in React.
-
-{% img blog/graphql-apollo-react/okta-app-settings.png alt:"Okta App Settings" width:"800" %}{: .center-image }
```

Comment:
Can you please delete this image too?

Reply:
I see it deleted?

Reply:
You deleted the line, but the file still exists in Git.